sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/CallMediaRecognizeSpeechOptions.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.callautomation.models;
+
+import com.azure.communication.common.CommunicationIdentifier;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/** The Recognize configurations specific for Continuous Speech Recognition. **/
+public class CallMediaRecognizeSpeechOptions extends CallMediaRecognizeOptions {
+    /*
+     * The length of end silence when user stops speaking and cogservice send
+     * response.
+     */
+    @JsonProperty(value = "endSilenceTimeoutInMs")
+    private Long endSilenceTimeoutInMs;
```

Comment:
according to azure board we want timeouts to be `Duration`. 
Please see [DtmfOptions](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/CallMediaRecognizeDtmfOptions.java) for reference.

Reply:
changed to duration